docs/configure_rum_android_sdk.md

```diff
@@ -171,12 +223,24 @@ To automatically track your views (activities, fragments, etc.), provide a track
 
 
 For instance, to set each fragment as a distinct view, use the following configuration in your [setup][1]:
-   
-```kotlin
-val configuration = Configuration.Builder(rumEnabled = true, ...)
-                 .useViewTrackingStrategy(FragmentViewTrackingStrategy(...))
-                 .build()
-```
+
+{{< tabs >}}
+{{% tab "Kotlin" %}}
+   ```kotlin
+       val configuration = Configuration.Builder(true, true, true, true)
+        .useViewTrackingStrategy(FragmentViewTrackingStrategy(...))
+        .build()
+   ```
+{{% /tab %}}
+{{% tab "Java" %}}
+   ```java
+       final Configuration configuration = new Configuration.Builder(true, true, true, true)
```

Comment:
I think we don't need to use `final`, since this variable is consumed instantly and not going to be captured by another context. wdyt?

It is used also in other Java examples.